server/sandbox_run.go

```diff
@@ -708,7 +708,17 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 			}
 			g.AddMount(mqueue)
 		}
-
+		if hostNetwork {
+			g.RemoveMount("/sys")
+			g.RemoveMount("/sys/cgroup")
+			sysMnt := spec.Mount{
+				Destination: "/sys",
+				Type:        "bind",
+				Source:      "/sys",
+				Options:     []string{"nosuid", "noexec", "nodev", "ro", "rbind"},
```

Comment:
I think we should check docker behavior here. Also, would privileged make this rw?

Reply:
the reason why I didn't make it rw is that when we use an userNS root in a container maps to (hopefully!) to a non-root user on the host and the user won't have write access to `/sys` when we bind mount it.

Reply:
but I'll check it out and see how Docker behaves

Reply:
@giuseppe Thanks! That sounds good.